terraform/iks-classic/README.md

```diff
@@ -0,0 +1,78 @@
+# auto-kubeflow
+Use Schematics/Terraform + Ansible to create a classic cluster on IBM Cloud,
+deploy kubeflow multi-user and integrate it with AppID service.
+
+The terraform templates are under this directory and used to create classic
+cluster as well as AppID instance. The kubeflow deployment and
+configuration are done by Ansible playbook under `ansible` directory.
+It uses [Ansible provisioner](https://github.com/radekg/terraform-provisioner-ansible)
+to perform Ansilbe playbook.
+
+Note: The Ansible playbook under `asnible` directory assumes the execution
+environment is Linux x86_64 architecture. If you want to run it locally, you may need
+to modify the playbook to accommodate your own environment.
+## Inputs
+
+| name | description | type | required | default | sensitive |
+| ------------------------- | ---------------------------------------------------------------------------------------------------------------------------------- | -------------- | ---------- | ------------------------------------ | ---- |
+|  ibmcloud_api_key      | IBM API key to to login ibm cloud and create resources |  string |  ✓  |      | ✓ |
+|  region                | The IBM Cloud regision you are going to create the resources | string  |   | us-south |  |
+|  org                   | IBM Cloud organization | string  |  ✓ |  |  |
+|  space                 | The space in IBM Cloud | string  | ✓ |  |  |
+|  resource_group        | The resource group in IBM Cloud | string  |  | default |  |
+|  kube_version          | kubernetes version | string  |  | 1.19.9 |  |
+|  appid_plan            | The plan for AppID | string  |  | lite |  |
+|  appid_name            | Instnace name for AppID service | string  |   | appid-instance |  |
+|  cluster_name          | Cluster name | string  |   | my-kfp-cluster | |
+|  cluster_worker_flavor | The machine flavor for the cluster worker nodes | string  |   | b3c.4x16 |  |
+|  cluster_worker_count  | number of workers in the cluster | number  |   | 2 |  |
+|  zone                  | The zone where the cluster will be created | string  |   | dal10 | |
```

Comment:
Does it support multi-zone clusters?

Reply:
nope, for multi-zone, need more customization for the vpc network